src/python/pants/backend/python/goals/setup_py.py

```diff
@@ -548,6 +550,20 @@ async def generate_setup_py_kwargs(request: GenerateSetupPyRequest) -> Finalized
         }
     )
 
+    if "long_description" not in setup_kwargs:
```

Comment:
I am wondering whether we should create a new class for the field as we did with the `LongDescriptionPath` to avoid literals and instead access `LongDescription.alias`?

Reply:
I don't think it's necessary. If we did want to do that, I'd recommend adding a class constant on `LongDescriptionPath`, like `LongDescriptionPath.KWARG`. But IMO that might be less clear, now readers need to jump to `LongDescriptionPath` to see what `KWARG` is.

Reply:
On a different note, I'm thinking that we should eagerly error if the `long_description_path` field is set and but you are already providing `long_description`. That's a likely bug the user might not realize. What do you think?

Some possible wording:

> InvalidFieldException(f"The {repr(LongDescriptionPath.alias)} field of the target {exported_target.target.address} is set, but 'long_description' is already provided explicitly in the `provides=setup_py()` field. You may only set one of these two values.")